docs/sync.js

```diff
@@ -182,6 +182,18 @@ class App {
    * @since v10.0.0
    */
   getApp(appId) {}
+
+  /**
+   * Adds a listener that will be fired on various events, including user login and logout
```

Comment:
Are we able to be more precise about which events?

Reply:
Specific events have been added.